pydle/utils/run.py

```diff
@@ -6,9 +6,7 @@
 
 def main():
     client, connect = _args.client_from_args('pydle', description='pydle IRC library.')
-    loop = asyncio.get_event_loop()
-    asyncio.ensure_future(connect(), loop=loop)
-    loop.run_forever()
+    asyncio.run(connect())
```

Comment:
This will drop 3.6 support.
I am not convinced this is worth it, given 3.6 is within 2 Ubuntu LTS releases of current(18.04 when 20.04 is current).

Reply:
For what it's worth, I'm currently needing to rewrite the IRC bot functionality in one of my infrastructure programs because the rest of the program now depends on 3.10. There's definitely a segment of people who update early, and I seem to have forcibly become part of them.

Reply:
My above comment aged poorly.
As Ubuntu 22.04 is now current (py 3.10), I am now OK with dropping python 3.6 (Ubuntu 18.04).